tables/zu-g1.utb

```diff
@@ -0,0 +1,15 @@
+# liblouis: isiXhosa/isiZulu braille code (grade 1)
+
```

Comment:
Needs copyright and license

Reply:
What happens if a file does not have a license?

Reply:
> What happens if a file does not have a license?

Then I think we are not allowed to distribute it

Reply:
OK then.

Reply:
Done